Resources/config/routing/security.xml

```diff
@@ -5,15 +5,15 @@
     xsi:schemaLocation="http://symfony.com/schema/routing http://symfony.com/schema/routing/routing-1.0.xsd">
 
     <route id="fos_user_security_login" path="/login" methods="GET POST">
-        <default key="_controller">FOSUserBundle:Security:login</default>
+        <default key="_controller">fos_user.security.controller:login</default>
```

Comment:
`The controller for URI "/login" is not callable. Expected method "login" on class "FOS\UserBundle\Controller\SecurityController", did you mean "loginAction"?`
and on all other bundle urls.
This should be `fos_user.security.controller:loginAction`?

Reply:
You're 100% right! I messed that up :). Fixed now!